src/pages/result/result.interfaces.ts

```diff
@@ -23,6 +23,6 @@ export interface FilterValues {
 }
 
 export interface RootState {
-  resultReducer: Data;
+  result: Data;
```

Comment:
I would like to improve semantic here. Instead of `Data` to have `ResultStore` or `ResultState`

Reply:
Okay! It will be done!